### ISSUE TITLE
Current location

### ISSUE THREAD

**Issue by dhochman** [dhochman/Skatespot-Organizer#9]
Should the map fragment, when loaded, automatically zoom into the user's current location or should it simply provide the user with an easy way (i.e. a button) to zoom into their current location when they want? 

I vote for the latter. The reason is that, personally, I might search for a spot somewhere other than where I am currently, zoom into its location on the map, then choose to check out the list view or detail view for a second, but then decide to look at the map view one last time. However, after navigating away from the map and then back to it, the map shows my current location, not the spot I was previously look at.

I think we should have a button that lets the user zoom into their current location whenever they want, but should otherwise have the map reload to where it was previously so that a user can freely toggle between different views without "losing" the spot they're interested in.

**Comment by tryn2hard** [comment]
Agreed. The map should always stay at the position the user selected, and only go to the current location when the user wants it.
